src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -55,78 +59,165 @@
-          journey: 'TEST_Registration',
-          match: [
-            '#/service/TEST_Registration',
-            '?journey=TEST_Registration',
-            '#/service/Registration',
-            '?journey=Registration',
-          ],
+        content: {
+          ...content,
+          alreadyHaveAnAccount: `Already have an account? <a href="?journey=TEST_Login">Sign in here!</a>`,
         },
-      },
-      links: {
-        termsAndConditions: 'https://www.forgerock.com/terms',
-      },
-      style: {
-        labels: 'floating',
-        showPassword: showPasswordParam,
-        logo: {
-          dark: '/img/fr-logomark-white.png',
-          light: '/img/fr-logomark-black.png',
+        journeys: {
+          forgotPassword: {
+            journey: 'TEST_ResetPasword',
+            match: [
+              '#/service/TEST_ResetPassword',
+              '?journey=TEST_ResetPassword',
+              '#/service/ResetPassword',
+              '?journey=ResetPassword',
+            ],
+          },
+          forgotUsername: {
+            journey: 'TEST_ForgottenUsername',
+            match: [
+              '#/service/TEST_ForgottenUsername',
+              '?journey=TEST_ForgottenUsername',
+              '#/service/ForgottenUsername',
+              '?journey=ForgottenUsername',
+            ],
+          },
+          login: {
+            journey: 'TEST_Login',
+            match: [
+              '#/service/TEST_Login',
+              '?journey',
+              '?journey=TEST_Login',
+              '#/service/Login',
+              '?journey',
+              '?journey=Login',
+            ],
+          },
+          register: {
+            journey: 'TEST_Registration',
+            match: [
+              '#/service/TEST_Registration',
+              '?journey=TEST_Registration',
+              '#/service/Registration',
+              '?journey=Registration',
+            ],
+          },
         },
-        sections: {
-          header: false,
+        links: {
+          termsAndConditions: 'https://www.forgerock.com/terms',
         },
-      },
-    });
+        style: {
+          labels: 'floating',
+          showPassword: showPasswordParam,
+          logo: {
+            dark: '/img/fr-logomark-white.png',
+            light: '/img/fr-logomark-black.png',
+          },
+          sections: {
+            header: false,
+          },
+        },
+      });
+    }
     new Widget({ target: widgetEl });
+    if (initializePingProtectEarly) {
+      try {
+        await protect.start({
+          envId: initializePingProtectEarly,
+          behavioralDataCollection: pauseBehavioralData === 'true',
+          consoleLogEnabled:
+            initializePingProtectEarly && initializePingProtectEarly?.length !== 0 ? true : false,
+        });
+        await protect.getData();
+        protect.pauseBehavioralData();
+        protect.resumeBehavioralData();
+      } catch (err) {
+        console.error(err);
+      }
```

Comment:
Are you just calling all of these for test purposes? Plus, won't the error condition never happen because you've wrapped all the underlying Protect methods in their own try-catches?

Reply:
Yeah, i was calling for test purposes. I can remove the try catch